conductor-client/src/main/resources/mail/templates/shared/ReentryTaskAlertTemplate.mustache

```diff
@@ -0,0 +1,79 @@
+<!doctype html>
+<html>
+<head>
+    <title>Reentry Case Management - Task{{subtitle}}</title>
```

Comment:
`Task{{subtitle}}` won't this mean the `Task` and whatever follow are right next to each other? Is that intentional?

Reply:
ah ya, it is actually intentional. if there is a subtitle on the entity, then this subtitle variable will look like ``: ${subtitle}``.